src/components/gataca-qrws/gataca-qrws.tsx

```diff
@@ -20,6 +20,7 @@ import {
 import { Success } from "./components/success/Success";
 import { RetryButton } from "./components/retryButton/RetryButton";
 import { QR } from "./components/qr/QR";
+import { ReadQR } from "./components/readQR/ReadQR";
```

Comment:
question [non-blocking]: instead of repeating the components, couldn't we just have one common and import them?